spring-cloud-sleuth-instrumentation/pom.xml

```diff
@@ -127,6 +127,11 @@
 			<artifactId>spring-cloud-starter-gateway</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-task</artifactId>
+			<optional>true</optional>
```

Comment:
In this case it means that task must be built before Sleuth.  I think we are good shape on this, but just wanted to capture the thought.

Reply:
We already require task to be built before Sleuth